components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/docs/examples/applications/applications.yaml

```diff
@@ -0,0 +1,13 @@
+delete:
+  - id: deleteApplication
+  - request:
+      method: DELETE
+      url: https://localhost:9443/api/am/admin/v1/applications/367a2361-8db5-4140-8133-c6c8dc7fa0c4
+      headers: |
+        Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8
+    response:
+      status:
+        code: 200
+        msg: OK
+      headers: |
+        Content-Type: application/json
```

Comment:
Don't we need a new line eof?

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/9604/commits/65c3eb3ee90be6b18f554501b649a7bcfa4e3473

Reply:
Thanks